src/installer/tests/HostActivation.Tests/FrameworkResolution/MultipleHives.cs

```diff
@@ -62,19 +55,6 @@ public void FrameworkHiveSelection(string requestedVersion, string tfm, bool? mu
                 .ShouldHaveResolvedFramework(MicrosoftNETCoreApp, resolvedVersion);
         }
 
-        [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)] // Multiple hives are only supported on Windows.
-        public void FrameworkHiveSelection_CurrentDirectoryIsIgnored()
```

Comment:
The tests in this file aren't actually using .NET 5. They check that we parse and compare the TFM and version correctly to resolve frameworks.

I think we still want this test case - pre-7.0, multi-level lookup enabled, current directory ignored. You should be able to just switch `Constants.Tfm.Net5` -> `Constants.Tfm.Net6`.

Reply:
Got it.